### PR TITLE
New Crowdin updates

### PR DESCRIPTION
**Pull request by abulte**

### CODE REVIEW

**Comment by maudetes** [comment]
Update `.po` files following #2645.

Added some missing translations (some about schemas, if you want to proofread these later @geoffreyaldebert).

**Review by maudetes — APPROVED** [review]
I updated the changelog because it is a huge translation update.

I'll let you merge this PR if it seems ok @abulte.

File: CHANGELOG.md
Comment (abulte) [inline CHANGELOG.md:10]:
```suggestion
- Update translations following frontend refactoring (a lot of translations have been moved to udata-gouvfr) [#2648](https://github.com/opendatateam/udata/pull/2648)
```

**Pull request merged**
